rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -326,6 +326,10 @@ def test_upgrade_v4_v5(globaldb):
             key_parts=[GeneralCacheType.CURVE_POOL_TOKENS],
         )
         assert len(pool_tokens_in_global_db) > 0, 'There should be some pool tokens set'
+        contracts_before_upgrade = cursor.execute(
```

Comment:
you can check that the name column is in the DB before the upgrade. And then check that the name column does not exist after